docs/how-to/how-to-use-virtio-mem-with-kata.md

```diff
@@ -41,13 +35,35 @@ Use following command to enable memory overcommitment of a Linux kernel.  Becaus
 $ echo 1 | sudo tee /proc/sys/vm/overcommit_memory
 ```
 
-Use following command start a Kata Container.
+Use following command start a Kata Container with containerd.
```

Comment:
```suggestion
Use following command to start a Kata Container.
```

This shouldn't be specific to containerd, right?

Reply:
Updated.